### ISSUE TITLE
Issue while training

### ISSUE THREAD

**Issue by abhiML** [meijieru/crnn.pytorch#19]
Start val
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: accredits           
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: cabral              
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: esters              
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: indulgent           
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: typologies          
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: bulwarks            
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: insensitivity       
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: fiche               
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: elephantiasis       
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: niamey              
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: divorces            
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: uncanny             
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: nameable            
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: colds               
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: hydrofoils          
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: eccentrically       
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: strictly            
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: misappropriate      
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: televisions         
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: turgid              
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: undefined           
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: margaritas          
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: notifying           
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: amniocentesis       
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: lands               
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: cunard              
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: camshaft            
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: taps                
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: mediated            
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: flexes              
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: plumier             
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: extradite           
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: paramountcy         
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: korans              
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: compel              
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: homburg             
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: noncommunicable     
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: fireguard           
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: servants            
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: dinnertime          
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: elopes              
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: innervate           
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: whys                
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: unhurt              
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: screechier          
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: chops               
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: saprophytes         
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: cannily             
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: laborers            
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: inns                
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: loused              
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: agrees              
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: gusty               
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: prizewinners        
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: cremation           
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: notoriously         
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: awesome             
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: musicals            
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: cuddliest           
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: unsupervised        
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: villus              
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: daters              
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: acheson             
qqqqqqqqqqqqqqqqqqqqqqqqqq => q                   , gt: unmentioned         
Test loss: inf, accuray: 0.000000

**I get this output. It doesn't change. The loss is infinite and the accuracy is zero. Is there some parameter I should tweak?**

**Comment by yisongbetter** [comment]
I have similar result with yours, could you please tell me how to solve this? the test loss decrease with iters, but accuracy is always 0

**Comment by Aurora11111** [comment]
me too ! @abhiML @yisongbetter  have you found solution?

**Comment by Aurora11111** [comment]
@abhiML I increase the datasets ,Now I work it well!
